client/components/notice/notice-action.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent, ReactNode } from 'react';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	'aria-label': string;
+	href: string;
+	onClick: () => void;
```

Comment:
Ideally, we shouldn't default to noop functions if it's not necessary. It seems like defaulting to nothing will do the same job, but will spare us from declaring an extra noop function and executing it needlessly. That being said, should `onClick` be typed as an optional prop(with `?`)?